src/interpolation.jl

```diff
@@ -85,35 +85,69 @@ function interpolate_kpoint(data_in::AbstractVecOrMat, kpoint_in::Kpoint, kpoint
     data_out
 end
 
+"""
+Compute indices of the vectors in G_vectors(basis_in[ik]) that are also present
+in G_vectors(basis_out[ik]).
+"""
+function transfer_blochwave_mapping(ik, basis_in::PlaneWaveBasis{T},
+                                    basis_out::PlaneWaveBasis{T}) where T
+    # Get indices of the G vectors of the old basis inside the new basis.
+    idcsk_out = index_G_vectors.(Ref(basis_out), G_vectors(basis_in.kpoints[ik]))
+    # In the case where G_vectors(basis_in.kpoints[ik]) are biggers than vectors
+    # in the fft_size box of basis_out, we need to filter out the "nothings" to
+    # make sure that the indices linearization works. It is not an issue to
+    # filter these vectors as this can only happen if Ecut_in > Ecut_out.
+    filter!(!isnothing, idcsk_out)
+    # Linearize the indices
```

Comment:
That comment adds no info besides the next code line. Might as well remove it (I know I had it too, but it's not really needed).